Dockerfile

```diff
@@ -4,7 +4,9 @@ WORKDIR /web
 
 ADD . ./
 
+RUN apk add curl
+
 RUN npm ci
 RUN npm run tailwind
 
-CMD [ "npm", "run", "dev"]
+CMD [ "npm", "start"]
```

Comment:
This was intentionally `npm run dev`. If you want to use docker in production, I'd be open to having a `Dockerfile.local` and `Dockerfile.production`, though.

Reply:
Oh, I understood. You mean this Dockerfile is for develop on local?
I'll fix.

Reply:
Yeah, that's what I've been doing. My deploy server just has all the dependencies installed, so I've had no need for docker in production, but I will gladly support it if people are deploying their own copies.